Source/Core/Core/PowerPC/JitCommon/Jit_Util.cpp

```diff
@@ -1058,7 +1058,7 @@ void EmuCodeBlock::SetFPRF(Gen::X64Reg xmm)
 void EmuCodeBlock::JitGetAndClearCAOV(bool oe)
 {
 	if (oe)
-		AND(8, PPCSTATE(xer_so_ov), Imm32(~XER_OV_MASK)); //XER.OV = 0
+		AND(8, PPCSTATE(xer_so_ov), Imm8(~XER_OV_MASK)); //XER.OV = 0
```

Comment:
AGP runs into a CPU bug here.  It's unrelated to the AGP, but needs to be stuffed into a PR somewhere.